crypto/ledger/ledger_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build ledger && test_ledger_mock
```

Comment:
It should only be the ledger build flag. Otherwise, it won't run with a real ledger.
So `test_ledger_mock` should be removed.